src/modules/ROOT/pages/changelog/202309/01.json

```diff
@@ -0,0 +1,89 @@
+{
+  "meta": {
+    "releasedOn": "2023.12.01"
+  },
+
+  "added": [
+    {
+      "what": [
+        "the options to clone and copy a page block within a tabbed page block."
+      ],
+      "why": [
+        "to provide an easier way to clone and copy tabbed page blocks."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1527"
+      ]
+    },
+    {
+      "what": [
+        "a dropdown to select the number of items to be displayed per page in a list of resources."
+      ],
+      "why": [
+        "to provide the user with the option to select how many resources can be displayed per page."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1534"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "the error message to be more verbose when user includes invalid or missing record input required values in a workflow function."
+      ],
+      "why": [
+        "to provide a clearer message on what and where the issue is."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1571"
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "undefined property error in the developer console after adding a tabs page block without providing a tab in the configuration."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1569"
+      ]
+    },
+    {
+      "what": [
+        "triggering of unsaved changes message after user adds an additional metric to generic chart and saves the chart."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1572"
+      ]
+    },
+    {
+      "what": [
+        "module field label and description shown on the same line when viewing a record on a public page."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1576"
+      ]
+    },
+    {
+      "what": [
+        "adding a translation to the field inputs `hint view` and `description view` toggles the visibility of `hint edit` and `description edit` input fields."
```

Comment:
Actually it was caused by changing any field translation. So can be reworded to 
`field hint and description toggled on for edit modes when updating field translations`